plugins/SkyCultureMaker/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+# This is the cmake config file for the Sky Culture Maker plugin
+SET(SCM_VERSION "0.1.0")
+
+
+
+# Option to manually control converter, defaults to TRUE, but overridden by Qt version
+OPTION(SCM_SHOULD_ENABLE_CONVERTER "Attempt to enable Sky Culture Converter" TRUE)
+SET(SCM_CONVERTER_ENABLED FALSE) # Default to disabled
+
+IF(SCM_SHOULD_ENABLE_CONVERTER AND NOT (QT_VERSION_MAJOR EQUAL "5"))
+  SET(SCM_CONVERTER_ENABLED TRUE)
+  MESSAGE(STATUS "Sky Culture Converter will be enabled.")
+
+  # On macOS, Homebrew is a common way to install dependencies.
+  # We should look for it in the default locations for Apple Silicon and Intel macs
+  # to find the correct version of libtidy. Otherwise a system library might be used.
+  IF(APPLE)
+    LIST(APPEND CMAKE_PREFIX_PATH "/opt/homebrew" "/usr/local")
+  ENDIF()
+
+  # download CPM.cmake
```

Comment:
But CPM is included already in main CMakeLists.txt file - why it needed here again?